lexicons/bsky.app/declaration.json

```
@@ -0,0 +1,16 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.declaration",
```

I think that a `"description"` may be especially useful here for self-documentation, particularly because the term "declaration" is quite generic.  Alternately, we might consider a more verbose name like `accountDeclaration`.
```suggestion
  "id": "app.bsky.declaration",
  "description": "Context for an account that is considered intrinsic to it. A declaration should be treated as immutable.",
```

---

good point 👍 
especially important on a schema like this, but as paul noted, we should be adding descriptions/comments to all of our schemas